pkgs/applications/networking/cisco-packet-tracer/customRequireFile.nix

```diff
@@ -0,0 +1,28 @@
+{ requireFile }:
+
+{ filename
+, version
+, alternativeURL
+, sha256
+}: requireFile {
+  name = filename;
+  message = ''
+    Unfortunately, we cannot download the file ${filename} automatically.
+    You have two known options to get this file:
+    - Download the deb file from the NetAcad website: http://www.netacad.com/
+      - You should download the exact same version (${version}) used in this package if it's not the exact version it will not pass.
```

Comment:
For brevity: `Download the exact same version (${version}) used in this package.`

Reply:
Suggestion:
```
The file '${filename}', version ${version}, is manually downloaded.
You may download it from https://www.netacad.com (or from other sources).

Once downloaded, check hash:
    sha256sum ${filename}
It should be: ${sha256}

Import it to your nix-store executing the command:
    nix-store --add-fixed sha256 ${filename}

Try to install the package again.
```